scss/components/shellbar.scss

```diff
@@ -289,4 +289,62 @@ $block-product-menu: #{$fd-namespace}-product-menu;
       margin-right: 8px;
     }
   }
-}
\ No newline at end of file
+}
+
+
+$fd-product-switcher-width: 450px;
+$fd-product-switcher-product-width: 140px;
+$fd-product-switcher-product-height: 40px;
+
+/*!
+.fd-product-switcher
+*/
+$block-product-switcher: #{$fd-namespace}-product-switcher;
+
+
+.#{$block-product-switcher} {
+  position: relative;
+  display: inline-block;
+
+  &__body {
+    padding: 20px 10px 0 10px;
+    color: initial;
+    > nav {
```

Comment:
I thought we were trying to avoid using direct element selectors for all of these components 🤔

Reply:
This will be fixed when we handle shellbar - I think this is just the stop gap so the docs site examples don't break. We did the same with product-menu.